annotationPipeline/pom.xml

```diff
@@ -58,4 +58,14 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <!-- required to build an executable jar -->
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
```

Comment:
Building the jar might work without adding the plugin in this module (it's already in the parent). It doesn't hurt to add it though afaik, just makes it easier to configure at the module level in the future if we want. It was only really necessary for the `annotator` module b/c that one would no longer work with the spring boot upgrade